lib/index.js

```diff
@@ -26,8 +26,17 @@
  *   This can be used to ship a processor with your package, to be used if no
  *   processor is found locally.
  *   If this isn’t passed, a warning is shown if `processorName` can’t be found.
+ * @property {string} [configurationSection]
+ *   This option will be used to give the client a hint of which configuration
+ *   section to use.
+ *   For example VSCode extensions use this to pick only settings that use this
+ *   as a prefix in order to prevent conflicts and reduce the amount of data
+ *   sent to the language server.
  *
  * @typedef {EngineFields & LanguageServerFields} Options
+ *
+ * @typedef UnifiedLanguageServerSettings
```

Comment:
Should have a description explaining that this is a schema matching (part of) editor config files?

Reply:
I added a description and a readme section.